json/src/main/java/com/basistech/rosette/dm/jackson/KnownAttribute.java

```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright 2017 Basis Technology Corp.
+* Copyright 2019 Basis Technology Corp.
```

Comment:
do we usually use 2017-2019? Does it matter?

Reply:
Usually we do, but our Checkstyle configuration for open source stuff only allows one year. I think I’ve read that if you have to pick one year for a copyright statement, it should be the first year it was copyrighted; but I’ve also read that since copyright is automatic it doesn’t really matter.